tests/settings.py

```diff
@@ -26,6 +26,7 @@
     'django.contrib.staticfiles',
     'django.contrib.admin',
     'django_freeradius',
+    'openwisp_utils.admin_theme',
```

Comment:
Check the import order. Make sure you see the admin theme instead of default django admin page.